neurolib/models/bold/model.py

```diff
@@ -67,19 +66,8 @@ def run(self, activity, append=False):
             * self.dt
         )
 
-        if self.BOLD.shape[1] == 0:
-            # add new data
-            self.t_BOLD = t_BOLD_resampled
-            self.BOLD = BOLD_resampled
-        elif append is True:
-            # append new data to old data
-            self.t_BOLD = np.hstack((self.t_BOLD, t_BOLD_resampled))
-            self.BOLD = np.hstack((self.BOLD, BOLD_resampled))
-        else:
-            # overwrite old data
-            self.t_BOLD = t_BOLD_resampled
-            self.BOLD = BOLD_resampled
-
```

Comment:
You seem to have the `append=True` case, why?

Reply:
The functionality for saving outputs is already in `models/model.py`.